python/tk_multi_loader/model_publishhistory.py

```diff
@@ -100,6 +100,33 @@ def async_refresh(self):
     ############################################################################################
     # subclassed methods
 
+    def _item_created(self, item):
+        """
+        Called when an item is created, before it is added to the model.
+
+        Override default behavior to only download thumbnails for the fields
+        we're interested in, and avoid downloading preview images.
+
+        :param item: The item that was just created.
+        :type item: :class:`~PySide.QtGui.QStandardItem`
+        """
+        # as per docs, call the base implementation
```

Comment:
Would be better to mention that we call the implementation of the base of `ShotgunModel` since we want to replace totally what `ShotgunModel` does.